omnibus/files/private-chef-ctl-commands/rotate_credentials.rb

```diff
@@ -122,6 +137,77 @@
   exit(0)
 end
 
+add_command_under_category "require-credential-rotation", "credential-rotation", "Disable the Chef Server and require credential rotation", 2 do
```

Comment:
We're traditionally had 1 ctl command per file, although I don't know how religiously we've stuck to that. While all the commands here are related to credentials, it might be best to separate our require-crediential-rotation, since it's in a different bucket from the others and doesn't seem to share much, if any code.